gui/default/index.html

```diff
@@ -29,786 +29,821 @@
-            <li class="divider" aria-hidden="true"></li>
-            <li><a href="" ng-click="advanced()"><span class="fas fa-fw fa-cogs"></span>&nbsp;<span translate>Advanced</span></a></li>
-            <li><a href="" ng-click="logging.show()"><span class="far fa-fw fa-file-alt"></span>&nbsp;<span translate>Logs</span></a></li>
-            <li class="divider" aria-hidden="true" ng-if="config.gui.debugging"></li>
-            <li><a href="/rest/debug/support" target="_blank" ng-if="config.gui.debugging"><span class="fa fa-user-md"></span>&nbsp;<span translate>Support Bundle</span></a></li>
-          </ul>
-        </li>
-      </ul>
-    </div>
-  </nav>
-
-  <div class="container" id="content">
-
-    <!-- Panel: Open, no auth -->
+  <noscript>
+    <nav class="navbar navbar-top navbar-default" role="navigation">
+      <div class="container">
+        <span class="navbar-brand" aria-hidden="true">
+          <img class="logo hidden-xs" src="assets/img/logo-horizontal.svg" height="32" width="117" alt=""/>
+          <img class="logo hidden visible-xs" src="assets/img/favicon-default.png" height="32" alt=""/>
+        </span>
+      </div>
+    </nav>
 
-    <div ng-if="openNoAuth" class="row">
-      <div class="col-md-12">
-        <div class="panel panel-danger">
-          <div class="panel-heading">
-            <h3 class="panel-title">
-              <div class="panel-icon">
-                <span class="fas fa-exclamation-circle"></span>
-              </div>
-              <span translate>Danger!</span>
-            </h3>
-          </div>
-          <div class="panel-body">
-            <p>
-            <span translate>The Syncthing admin interface is configured to allow remote access without a password.</span>
-            <b><span translate>This can easily give hackers access to read and change any files on your computer.</span></b>
-            <span translate>Please set a GUI Authentication User and Password in the Settings dialog.</span>
-            </p>
-          </div>
-          <div class="panel-footer">
-            <button type="button" class="btn btn-sm btn-default pull-right" ng-click="showSettings()">
-              <span class="fas fa-cog"></span>&nbsp;<span translate>Settings</span>
-            </button>
-            <div class="clearfix"></div>
+    <div class="container content">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="panel panel-danger">
+            <div class="panel-heading">
+              <h3 class="panel-title">
+                <div class="panel-icon">
+                  <span class="fas fa-exclamation-circle"></span>
+                </div>
+                <span translate>Warning!</span>
+              </h3>
+            </div>
+            <div class="panel-body">
+              <p>
+              <span translate>The Syncthing admin interface requires JavaScript to function correctly. Please enable JavaScript in your Web browser and try again.</span>
```

Comment:
```suggestion
              <span>The Syncthing admin interface requires JavaScript. Please enable JavaScript in your Web browser and try again.</span>
```

Reply:
(no translation is going to happen here because no javascript, and slightly shorter.)